website/docs/cloud-docs/api-docs/changelog.mdx

```diff
@@ -9,6 +9,19 @@ description: >-
 
 Keep track of changes to the API for Terraform Cloud and Terraform Enterprise.
 
+## 2023-05-25
+
+* Add the new `enforcement-level` property in the request and response of [Policies endpoints](/terraform/cloud-docs/api-docs/policies).
+* Deprecate the old `enforce` property in the request and response of [Policies endpoints](/terraform/cloud-docs/api-docs/policies).
+* Update the [entitlement set](/terraform/cloud-docs/api-docs/organizations#show-the-entitlement-set), [feature set](/terraform/cloud-docs/api-docs/feature-sets#sample-response) and [subscription](/terraform/cloud-docs/api-docs/subscriptions#sample-response) endpoints with the following properties, which are for Run Task and Policy limits in the free tier;
```

Comment:
The following feels clearer to me, what do you think?
```suggestion
* Add new properties to limit run tasks and policies for the Terraform Cloud free tier. We updated the [entitlement set](/terraform/cloud-docs/api-docs/organizations#show-the-entitlement-set), [feature set](/terraform/cloud-docs/api-docs/feature-sets#sample-response), and [subscription](/terraform/cloud-docs/api-docs/subscriptions#sample-response) endpoints with the following properties:
```

Reply:
Looks good. changed.